src/modules/services/mysql.nix

```
@@ -177,6 +181,17 @@ in
       '';
     };
 
+    useDefaulsExtraFile = lib.mkOption {
+      type = types.nullOr types.bool;
+      default = null;
```

```suggestion
      default = true;
```

to keep it as before 🤔

---

That would be `false` then.

---

Updated it.